app/assets/javascripts/shared/inline_editing.js

```diff
@@ -80,7 +81,7 @@ var inlineEditing = (function() {
         }
 
         container.find('.view-mode')
-          .html(viewData)
+          .text(viewData)
```

Comment:
For smart annotation fields, we still need to use html. You can check if the field for smart annotation `container.data('smart-annotation')`. As I remember now we use it in comments.